index.js

```diff
@@ -48,6 +48,15 @@ module.exports.dlp = {
   v2beta1: googleProtoFiles('privacy/dlp/v2beta1/dlp.proto'),
 };
 
+module.exports.embedded_assistant = {
```

Comment:
Do you mind casing it `embeddedAssistant` instead to match more conventional JS style? The other names in the repo use all lowercase, e.g. `module.exports.storagetransfer` and `module.exports.videointelligence`. That seems strange to me, but if that's more preferable, feel free.

Reply:
Done